features/projects/components/loading-screen/loading-screen.style.ts

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+import { color } from "@styles/theme";
+
+export const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 8.063rem;
+`;
+export const Loader = styled.div`
+  border-width: 0.5rem;
+  border-style: solid;
+  border-color: ${color("primary", 50)} ${color("primary", 50)}
+    ${color("primary", 50)} ${color("primary", 600)};
+  width: 3.625rem;
+  height: 3.625rem;
+  border-radius: 50%;
+  position: relative;
+  -webkit-animation: spin 2s infinite;
```

Comment:
Is this required? Typically these kind of prefixes are added by styled-components automatically

Reply:
Is the prefix that -webkit you're referring to? I'm not familiar with the jargon yet. 😅